hil/cli/node.py

```diff
@@ -157,6 +157,13 @@ def node_power_cycle(node):
     client.node.power_cycle(node)
 
 
+@node_power.command(name='status')
+@click.argument('node')
+def node_power_status(node):
+    """Returns node power status"""
+    print client.node.power_status(node)
```

Comment:
Let's put parens around the print() call, for python 3 compat (at some point we should actually make a push on improving that situation, but let's not make it worse).

Reply:
cool, done.